parameters/example_population_params.json

```diff
@@ -88,12 +88,13 @@
             "pSingleOlder":0.14
         },
         "buildingDistribution":{
-            "populationToHospitalsRatio":10000,
+            "populationToHospitalsRatio":20000,
```

Comment:
I am sure that I have misunderstood someting here, but why change this populationToHospitalRatio? The thinking was thta every fifth hospital would be a Covid hospital, but I don't understand why to change this ratio here

Reply:
Hi Paul, I thought you said in an email to change this to 1:20000? I've probably misread it. I can revert this easy enough

Reply:
It was probably my mistake, let's kepe it to 1:10000 for now